code/go/0chain.net/blobbercore/handler/object_operation_handler.go

```diff
@@ -520,7 +519,7 @@ func (fsh *StorageHandler) CommitWrite(ctx context.Context, r *http.Request) (*b
 	}
 
 	db.Delete(connectionObj)
-	common.DeleteConnectionObjEntry(connectionID)
+	allocation.DeleteConnectionObjEntry(connectionID)
```

Comment:
Since there can be many lock queues it might be better to run it in goroutine and avoid making client to wait.